hypothesis_bio/hypothesis_bio.py

```diff
@@ -176,44 +177,112 @@ def fasta(draw) -> str:
 
 
 @composite
-def fastq_quality(draw: Callable, size=0) -> str:
+def sequence_id(
+    draw, blacklist_characters: Sequence[str] = ">@", max_size: int = 100
```

Comment:
Is there a reason that the sequence ID can't contain one of those characters?

> The original Sanger FASTQ format was by no means perfect. The ‘@’ and ‘+’ characters have dual usage as line markers or anywhere within the quality string. Simple indexing of the file looking for lines starting with ‘@’ is therefore not possible.

\- _The Sanger FASTQ file format for sequences with quality scores, and the Solexa/Illumina FASTQ variants_

Reply:
Not explicitly. In general, I guess most people don't use these characters in their sequence IDs as they are usually the header markers. I added it as an option though. Again, I have no strong preference, so happy to change it

Reply:
I think it would make more sense to make it `None` for this